src/Common/RJSF/templates/ButtonTemplates/RemoveButton.tsx

```diff
@@ -20,7 +20,7 @@ import { IconButtonProps } from '@rjsf/utils'
 import { ReactComponent as CrossIcon } from '../../../../Assets/Icon/ic-cross.svg'
 
 export const RemoveButton = ({ icon, iconType, registry, uiSchema, ...props }: IconButtonProps) => (
-    <button {...props} type="button" className="dc__outline-none-imp p-0 dc__transparent flex cursor" title="Remove">
+    <button {...props} type="button" className="dc__outline-none-imp p-0 dc__transparent scn-6 flex cursor" title="Remove">
```

Comment:
Why are we adding scn-6?

Reply:
instead of fcr-5 we know need scn-6 as per design decisions of product team